test/acceptance/features/volumeMount.feature

```diff
@@ -0,0 +1,166 @@
+Feature: Bind an application to a service using volume mount
+
+    As a user of Service Binding Operator
+    I want to bind applications to services it depends on
+    using volume mount
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+
+    Scenario: Binding is injected as file into application pod at given location
```

Comment:
Let be more specific about the scenario setup: "a given location" is actually location referred by "SERVICE_BINDING_ROOT` env var declared on the application deployment.

Reply:
Done. 172198aa20026266b5b017693826e754be0f1567